sample-operators/webpage/src/main/java/io/javaoperatorsdk/operator/sample/WebPageStandaloneDependentsReconciler.java

```diff
@@ -85,7 +87,7 @@ private void createDependentResources(KubernetesClient client) {
     Arrays.asList(configMapDR, deploymentDR, serviceDR, ingressDR).forEach(dr -> {
       dr.setKubernetesClient(client);
       dr.configureWith(new KubernetesDependentResourceConfig()
-          .setLabelSelector(DEPENDENT_RESOURCE_LABEL_SELECTOR));
+          .setLabelSelector(SELECTOR + "=true"));
```

Comment:
Why was the tests working if the selector was wrong?

Reply:
since those were not mutal exclusive, the other selector is about not having a label what was true.